difftool/diff/diff_test.go

```diff
@@ -0,0 +1,13 @@
+package diff
+
+import (
+	"testing"
+
+	v1 "code.vegaprotocol.io/vega/protos/vega/events/v1"
+)
+
+func TestXxx(t *testing.T) {
+	tt := []*v1.Transfer{}
+	tt = append(tt, &v1.Transfer{Id: "12345"}, &v1.Transfer{Id: "67890"})
+	println(sliceToString(tt))
+}
```

Comment:
Does this still need to be in here? It looks like local testing code.

Reply:
but what a nice test it is!

Reply:
so elegant!

Reply:
removed